automation/common.sh

```diff
@@ -182,14 +182,11 @@ run_tests_storage() {
     create_loop_devices 16
```

Comment:
We don't use this now, and there is no need to change it. The only change needed is to delete the entire directory (#43).